tests/test_mongo_db_from_config.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env pytest -vs
+"""Tests for mongo_db_from_config."""
+# Standard Python Libraries
+import os
+
+# Third-Party Libraries
+import mongomock
+import pytest
+
+# cisagov Libraries
+import mongo_db_from_config
+
+# define sources of version strings
+RELEASE_TAG = os.getenv("RELEASE_TAG")
+PROJECT_VERSION = mongo_db_from_config.__version__
+
+
+@pytest.mark.skipif(
+    RELEASE_TAG in [None, ""], reason="this is not a release (RELEASE_TAG not set)"
+)
+def test_release_version():
+    """Verify that release tag version agrees with the module version."""
+    assert (
+        RELEASE_TAG == f"v{PROJECT_VERSION}"
+    ), "RELEASE_TAG does not match the project version"
+
+
+def test_db_from_config_bad_yaml_no_database_entry():
+    """Test that YAML parsing fails due to a missing database entry."""
+    with pytest.raises(KeyError) as e_info:
+        db_connection = mongo_db_from_config.db_from_config(
+            "no_database_configuration.yml"
+        )
+        assert (
+            db_connection.list_collection_names() == []
+        ), "Testing database should have no collections"
+    assert e_info.value.args[0] == "database", "Expected a missing 'database' key"
+
+
+def test_db_from_config_bad_yaml_no_db_uri():
+    """Test that YAML parsing fails due to a missing database uri entry."""
```

Comment:
I understand why you put "uri" in lower case here, but I think proper capitalization is justified if you are not quoting "uri" (as you do in the assertion below).  Feel free to ignore this if you feel strongly about it:
```suggestion
    """Test that YAML parsing fails due to a missing database URI entry."""
```

Reply:
Please see [4061322](https://github.com/cisagov/mongo-db-from-config/pull/8/commits/4061322fd641ae5974aaf75496c28a32c018081c).